assets/sass/style.sass

```diff
@@ -214,6 +214,8 @@ $colors: mergeColorMaps(("secondary": ($secondary, $white), "twitter-blue": ($tw
 .is-sticky
   +sticky
   top: $navbar-height + 3rem
+  height: 80vh
+  overflow-y: scroll
```

Comment:
Nice improvement. A few comments:

- Are you sure that you want to apply these changes to everything that "is-sticky" vs. only the left-side nav? (I'm not sure what else "is-sticky" is used for.)
- The nav content gets cropped
  > ![image](https://user-images.githubusercontent.com/4140793/83686231-dd61e500-a5b7-11ea-8069-2b5d61cffec4.png)